pkg/server/sandbox_run.go

```diff
@@ -190,13 +151,31 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 
 	sandboxLabels := buildLabels(config.Labels, containerKindSandbox)
 
+	var snapshotterOption containerd.NewContainerOpts
+	switch securityContext.GetNamespaceOptions().GetUser() {
+	case runtime.NamespaceMode_CONTAINER:
+		return nil, errors.New("unsupported user namespace mode: CONTAINER")
+	case runtime.NamespaceMode_NODE:
+		snapshotterOption = customopts.WithNewSnapshot(id, containerdImage)
+	case runtime.NamespaceMode_NODE_WIDE_REMAPPED:
+		fallthrough
```

Comment:
Not sure we want this option or not, though. For now, I'd not implement until talking with upstream. That option, IIUC, made sense when no containr runtime supported userns remapping, and now they do. So... maybe it is not important anymore?

Reply:
Ok... Meanwhile, I will keep the CRI gRPC changes, and return an error if `NamespaceMode_NODE_WIDE_REMAPPED` is used, so that it does not silently implement `POD` when the kubelet asks for `NODE_WIDE_REMAPPED`.